pkgs/development/libraries/qt-5/5.15/default.nix

```diff
@@ -329,9 +329,6 @@ let
         propagatedBuildInputs = [ self.qtbase.dev buildPackages.makeBinaryWrapper ]
           ++ lib.optional stdenv.isLinux self.qtwayland.dev;
       } ../hooks/wrap-qt-apps-hook.sh;
-    } // lib.optionalAttrs config.allowAliases {
-      # remove before 23.11
```

Comment:
@lelgenio any idea what was the purpose of this comment? Maybe @ttuegel ?

Reply:
According to `git blame` @jtojnar added this message.

Reply:
In https://github.com/NixOS/nixpkgs/pull/222928 .

Reply:
I added the alias and the comment when i converted the set to use makeScopeWithSplicing. The purpose was to remove the alias before 23.11 :P